doc/introduction/operations.rst

```diff
@@ -114,6 +114,24 @@ array([[ 0.92387953+0.j,  0.+0.j ,  0.-0.38268343j,  0.+0.j],
 [ 0.+0.j,  0.+0.38268343j,  0.+0.j,  -0.92387953+0.j]])
 
 
+Matrix to Operator functions
```

Comment:
is there a reason we don't highlight the feature where it returns a pauli sentence instead here? might be nice because this section is about the different kinds of transformations you can do.

Tangentially, this section makes me think of `QubitUnitary` or similar Operators. Could they possibly tie in here too?

Reply:
> is there a reason we don't highlight the feature where it returns a pauli sentence instead here? might be nice because this section is about the different kinds of transformations you can do. It could be enough to just have it in the full function docstring, i'm just unsure what we want to advertise.

Yes we could add that feature in this section, since `PauliSentences` are mostly meant to be used internally, I don't think its super necessary to advertise it here. But I honestly don't mind if we add it in here at all!

> Tangentially, this section makes me think of `QubitUnitary` or similar Operators. Could they possibly tie in here too?

They are related, but this section is specifically for functions of operators and not operators themselves, so I don't think putting `QubitUnitary` in here makes sense.

Reply:
- if it's meant to be internal, I also wouldn't add it here, nbd
- was not proposing to add QubitUnitary here per se for exactly that reason, but the line between a transform and an operator can be grey. wondering if they can use each other somehow 🤔

Reply:
Ahh, that's a good idea. I guess this is slightly different because it specifically expresses the matrix as a linear combination of pauli words, where as in the other case we just wrap the matrix in a PennyLane operator class so that I can be used with the rest of our pipeline. Qubit unitary requires the matrix to be unitary, while this function has no such constraints.  

I can't think of how we could combine them.